docs/quickstart.md

```diff
@@ -0,0 +1,87 @@
+
+1. Login to Google Cloud console and click on the Cloud shell button on the top right corner.
+
+    ![cloud shell](https://services.google.com/fh/files/misc/cloud_shell.png)
+
+2. This is launch a terminal on Cloud console. Wait for the terminal to load.
+
+    ![provision](https://services.google.com/fh/files/misc/provision.png)
+
+3. Once the terminal is loaded, run the following command:
+
+    ```sh
+    sudo apt-get install google-cloud-sdk-spanner-migration-tool
+    ```
+
+4. Wait for the installation to complete. Spanner migration tool can be accessed using the following gCloud CLI command:
+
+    ```sh
+    gcloud alpha spanner migrate <COMMAND>
+    ```
+
+## Launching the web UI for Spanner migration tool
+
+1. In order to launch the web UI for Spanner migration tool, run the following command:
+
+    ```sh
+    gcloud alpha spanner migrate web
+    ```
+
+    {: .important }
+    You will be asked to authorize running this command by providing your Google Cloud credentials. This allows Spanner migration tool to access resources on your behalf.
+
+2. This is launch the web UI in the Cloud shell instance, and show the following message in the terminal logs:
+
+    ```sh
+    Starting Spanner migration tool UI at: http://localhost:8080
+    ```
+
+3. **Without exiting the Cloud shell command**, click on the "Web preview" icon on the top right corner of the cloud shell terminal, and then click on "Preview on port 8080". This will open the Spanner migration tool UI in a new tab. The web address of the UI would something like `https://8080-cs-<random-string>.cs-<region>-vwey.cloudshell.dev/`.
+
+    ![web preview](https://services.google.com/fh/files/misc/web_preview.png)
+
+    {: .important }
+    Keep the Cloud console tab with the cloud shell running open in the background. Do not close this tab!
+
+## Connecting to a source database
+
+{: .important }
+This example describes connecting to a CloudSQL MySQL database using IP whitelisting. The list of steps here are generic and can be followed for any source instance.
+
+1. In a new tab on the Cloud shell terminal, run the following command to fetch the IP address of the Cloud shell instance
+
+    ```sh
+    curl ifconfig.me
+    ```
+
+2. Copy the returned IP address. Configure the IP address in the IP whitelisting section of the CloudSQL instance. Follow [this guide](https://cloud.google.com/sql/docs/mysql/configure-ip#add) to configure the IP whitelisting.
+
+3. Go back to the Spanner Migration Tool web UI launched earlier. Follow steps in the [connect to source](./ui/connect-source.md) page to connect to your MySQL instance.
```

Comment:
Add a comment- the same steps can be followed for any database which is accessible from Cloud shell.
Note: This can not be used for databases within a VPC